keyboards/heliar/wm1_hotswap/info.json

```diff
@@ -0,0 +1,366 @@
+{
```

Comment:
Would you please replace the contents of this file with those of the attached file? It simply updates the labels to those I posted on the `wm1_hotswap.h` file.

[info.json.txt](https://github.com/qmk/qmk_firmware/files/3884215/info.json.txt)

Reply:
I have updated the related info.json, please check it~